service/src/test/java/bio/terra/profile/pact/provider/BPMProviderTest.java

```diff
@@ -0,0 +1,137 @@
+package bio.terra.profile.pact.provider;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.when;
+
+import au.com.dius.pact.provider.junit5.HttpTestTarget;
+import au.com.dius.pact.provider.junit5.PactVerificationContext;
+import au.com.dius.pact.provider.junitsupport.Provider;
+import au.com.dius.pact.provider.junitsupport.State;
+import au.com.dius.pact.provider.junitsupport.loader.PactBroker;
+import au.com.dius.pact.provider.spring.junit5.PactVerificationSpringProvider;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.common.iam.AuthenticatedUserRequestFactory;
+import bio.terra.profile.app.Main;
+import bio.terra.profile.app.configuration.BeanConfig;
+import bio.terra.profile.app.controller.UnauthenticatedApiController;
+import bio.terra.profile.db.ProfileDao;
+import bio.terra.profile.service.crl.AzureCrlService;
+import bio.terra.profile.service.crl.GcpCrlService;
+import bio.terra.profile.service.iam.SamService;
+import bio.terra.profile.service.iam.model.SamResourceType;
+import bio.terra.profile.service.job.JobService;
+import bio.terra.profile.service.profile.exception.ProfileNotFoundException;
+import bio.terra.profile.service.profile.model.BillingProfile;
+import bio.terra.profile.service.status.ProfileStatusService;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.data.jdbc.JdbcRepositoriesAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.cache.CacheManager;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+
+@Tag("provider-test")
+@Provider("bpm-provider")
+@PactBroker
+// for local testing, put any test pacts in the service/pacts folder.
+// then comment out the above line, and uncomment the following line
+// @PactFolder("pacts")
```

Comment:
Huh, it's annoying pact can let you control this via an env var or some other config.

Reply:
there _may_ be a way, but I haven't found it yet